.github/workflows/main.yml

```diff
@@ -0,0 +1,41 @@
+name: Code Quality Check
+
+on:
+  push:
+    branches: [master]
+  pull_request:
+    branches: [master]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [12.x, 14.x]
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Use Node.js ${{ matrix.node-version }}
+        uses: actions/setup-node@v1
+        with:
+          node-version: ${{ matrix.node-version }}
+
+      - name: Install dependencies
+        run: yarn
+
+      - name: Check Code Styling
+        run: yarn prettier --ignore-path .gitignore --check .
```

Comment:
I'm not sure if we have prettier in this branch.

Reply:
true, removing